### ISSUE TITLE
Error: Font Not Found

### ISSUE THREAD

**Issue by daxliniere** [xsrf/easyeda-labelmaker#10]
Hey there,
I just installed your extension and as soon as I open it, I get "Error: Font Not Found". I looked on this Github page for what I was doing wrong and read about the Fonts Manager. Call me daft, but I can't find EasyEDA's Fonts Manager. I even googled for it and the only result I got was the page I first learned about it.
![image](https://github.com/xsrf/easyeda-labelmaker/assets/8109020/54df1101-cb0c-495a-b6fa-3cfb2def7fee)

Any ideas?

Cheers!
Dax.

**Comment by daxliniere** [comment]
I found the problem, but not sure how to solve it.
If you look carefully, you'll see that the included font, though selected, isn't loaded. Perhaps EE changed a setting to allow only certain filetypes to be imported as an extension?

![EE_Labels_bug](https://github.com/xsrf/easyeda-labelmaker/assets/8109020/1e5cef36-a478-4050-a909-0f63a4035efc)

**Comment by xsrf** [comment]
Have you tried opening the Fonts manager via `Label Maker` - `Manage Fonts` and then adding the font there?

**Comment by daxliniere** [comment]
Hi Andreas! Thanks for your reply.
Aha! I see that now. Thank you.

I tried refreshing font cache, but no dice and had to install the font manually. (no big deal)
On my laptop, it worked first time, for some reason!

All the best, Andreas.
-Dax.

**Comment by daxliniere** [comment]
Wait! No, I worked out what happened.
When I installed on my desktop PC, I must have downloaded this project's ZIP file, rather than from the Releases section and I've just noticed that the font isn't included in that ZIP file.
![image](https://github.com/xsrf/easyeda-labelmaker/assets/8109020/06a38e9e-217f-48a5-bc39-8c1f4df774b8)

When I took that screencap this evening, it was on my laptop (which worked first time, despite the font file not being shown in the import files list). That time I definitely downloaded it from the Releases page.

Sorry to have bothered you, Andreas!

All the best,
Dax.
